packages/atomic/src/components/insight/atomic-insight-facet/atomic-insight-facet.tsx

```diff
@@ -59,7 +53,6 @@ export class AtomicInsightFacet
   @InitializeBindings() public bindings!: InsightBindings;
   public facet!: InsightFacet;
   public searchStatus!: InsightSearchStatus;
-  public withSearch = false;
```

Comment:
The insight facet never had support for facet search. This is a relic of the past before the refactor of facet-common. 

Delete !!!! Less bundle size 😋

Reply:
Insight panel components exist only for the preview feature in the admin (ie: kinda like a demo if you will)

They are not "production ready" for customers, or actually in use to power an insight panel for end user. This might happen later this year, when the service team invest in it.

I would remove the import for now.